utils/supported-browsers-from-mdn.mjs

```diff
@@ -25,10 +26,29 @@ function getBrowsersFromFeature(feature) {
 		let version;
 
 		if (Array.isArray(browserSupport)) {
-			version = browserSupport.find(({ alternative_name }) => !alternative_name)?.version_added;
+			const versions = browserSupport.sort((a, b) => Number(a?.version_added) - Number(b?.version_added));
```

Comment:
Should we use something like `semver` from npm to parse these version strings?

Should give better results for `Safari 15` vs `Safari 15.1`

Reply:
While I thought about that I also thought that I wasn't sure how that was going to fare in terms of unexpected inputs and such. I did run a simple test in these lines:

`['8','1','2','1.1','6'].sort((a, b) => Number(a) - Number(b))`

Which yields `['1', '1.1', '2', '6', '8']` (that looks correct to me)

Reply:
That looks great!

At the moment there is no browser that is tracked in `MAJOR.MINOR.PATCH` only Safari and older Opera have `MAJOR.MINOR` which can be handled as floats.